src/pages/home/report/ReportActionItemCreated.tsx

```diff
@@ -119,6 +119,7 @@ export default withOnyx<ReportActionItemCreatedProps, ReportActionItemCreatedOny
             prevProps.report?.description === nextProps.report?.description &&
             prevProps.personalDetails === nextProps.personalDetails &&
             prevProps.policy?.description === nextProps.policy?.description &&
-            prevProps.report?.reportName === nextProps.report?.reportName,
+            prevProps.report?.reportName === nextProps.report?.reportName &&
+            prevProps.report?.errorFields === nextProps.report?.errorFields,
```

Comment:
@jayeshmangwani We should add the errorFields to memo function. If not, the error doesn't appear immediately because we don't clear personal detail in failure data, it leads `personalDetails` doesn't change.